bitnami/tomcat/templates/_pod.tpl

```diff
@@ -110,7 +110,7 @@ containers:
   {{- include "common.tplvalues.render" (dict "value" .Values.extraVolumeMounts "context" $) | nindent 2 }}
   {{- end }}
 {{- if .Values.sidecars }}
-{{- include "common.tplvalues.render" ( dict "value" .Values.sidecars "context" $) }}
+{{ include "common.tplvalues.render" ( dict "value" .Values.sidecars "context" $) }}
```

Comment:
Hi,
  Could you provide some comment on this change ?

Reply:
As i noted in #6501, sidecars have issue too, problem was that "-" removed previous whitespaces, include new line and template was generated like this:
```
        volumeMounts:
        - name: data
          mountPath: /bitnami/tomcat- image: bitnami/jmx-exporter:latest
        imagePullPolicy: IfNotPresent
        name: jmx-exporter
```